apps/meteor/app/authorization/server/methods/addPermissionToRole.ts

```diff
@@ -15,6 +16,10 @@ declare module '@rocket.chat/ddp-client' {
 
 Meteor.methods<ServerMethods>({
 	async 'authorization:addPermissionToRole'(permissionId, role) {
+		if (role === 'guest' && !AuthorizationUtils.hasRestrictionsToRole(role) && (await License.hasValidLicense())) {
+			AuthorizationUtils.addRolePermissionWhiteList(role, await License.getGuestPermissions());
+		}
```

Comment:
For some reason EE tests were failing on CI because we didn't have any restrictions stored for the guest role (which didn't happen locally) :man_shrugging: so I went through all of the places where we populate such restrictions and couldn't find anything off, please tell me if you do
So I decided to reinforce these restrictions in the endpoint/method itself (which I believe is harmless) and create another task to identify what could be causing this issue later